index.js

```diff
@@ -1,6 +1,14 @@
 /* eslint-env node */
 'use strict';
 
+const PLAID_LINK_SCRIPT_TAG = '<script src="https://cdn.plaid.com/link/v2/stable/link-initialize.js"></script>';
```

Comment:
do you know if there's a way to add a [subresource integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) attr to this tag?

Reply:
I like this idea. 

However, after discussion with the team, we decided it is risky because we do not own the Plaid JS, can't host it, and will not be updated when they change their code, which would break our code. So we're going to pass on this.